rosdep/python.yaml

```diff
@@ -5151,6 +5151,11 @@ python3-dev:
   openembedded: [python3@openembedded-core]
   rhel: ['python%{python3_pkgversion}-devel']
   ubuntu: [python3-dev]
+python3-distutils:
+  debian: [python3-distutils]
```

Comment:
This is only supported on buster or later, but we still support stretch for Melodic.  So please use the null syntax here to disable the stretch support.

Reply:
null syntax for stretch in fe9bde3